cclib/parser/serenityparser.py

```diff
@@ -45,3 +45,7 @@ def extract(self, inputfile, line):
 
         if line[5:21] == "Basis Functions:":
             self.set_attribute("nbasis", int(line.split()[2]))
+
+        # Extract dispersion correction
+        if "Dispersion Correction (" in line:
+            self.set_attribute("dispersionenergies", [float(line.split()[3])])
```

Comment:
```suggestion
            self.append_attribute("dispersionenergies", float(line.split()[3]))
```
because there will be one for each step of a geometry optimization, PES scan, etc.